style.css

```diff
@@ -1,554 +1,1111 @@
-  margin-bottom: 0;
-  margin-left: 0;
-  margin-right: 0;
-  width: 19.5rem;
-  border-radius: 0%;
-  border: none;
-  font-family: 'Inter', sans-serif;
-  font-size: 17px;
-  font-weight: 700;
-  line-height: 24px;
-  letter-spacing: 0.03em;
+.about-container div > button {
+  width: 40%;
+  height: 42px;
+  color: #fff;
+  background-color: #ff6b00;
+  margin-top: 3%;
+  border: 0;
   text-align: center;
-  color: var(--txtcolor);
-  background-color: var(--maincolor);
+  line-height: 2rem;
+  text-decoration: none;
 }
 
-.work .carttt > .cart > button:hover {
-  background: var(--btncol2);
-  border: 1px solid var(--maincolor);
-  color: var(--maincolor);
+.about-container div hr {
+  width: 100%;
+  margin-top: 100%;
+  color: #fff;
 }
 
-.work > .first-box > .imgsection > .ttwork > button {
-  background: var(--maincolor);
-  border: none;
-  width: 7rem;
-  padding: 0.5rem;
-  font-family: 'Inter', sans-serif;
-  color: var(--txtcolor);
-  font-weight: 400;
-  font-size: 1rem;
+.about-container .skill-section {
+  padding: 11px 24px 0 24px;
+  display: flex;
+  flex-direction: column;
+  background-image: none;
+  width: 100%;
+  height: 120vh;
 }
 
-.work > .first-box > .imgsection > .ttwork > button:hover {
-  background: var(--btncol2);
-  border: 1px solid var(--maincolor);
-  color: var(--maincolor);
+.about-container div > ul {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  margin: 3%;
+  width: inherit;
 }
 
-/* work carts selector */
-.work > .carttt > .cart {
-  display: flexbox;
-  flex-direction: column;
-  margin-bottom: 2rem;
-  padding-top: 2.5rem;
-  height: 20rem;
-  width: 19.5rem;
-  left: 0;
-  top: 0;
-  border: none;
-  border-radius: 0;
-  background: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url(Resources/Images/Mask-Group.png);
-  background-position: center center;
-  background-repeat: no-repeat;
-  color: var(--txtcolor);
+.about-container div li {
+  width: 100%;
+  text-align: center;
+  list-style: none;
+  margin: 2%;
+  height: 15px;
+  font-family: 'Inter', sans-serif;
+  font-size: 18px;
+  font-weight: 200;
 }
 
-.work > .carttt > .cart > h2 {
-  padding: 0.5rem 1rem;
-  font-weight: 500;
-  font-family: 'Crete Round', serif;
-  font-size: 32px;
-  line-height: 40px;
+.about-container div li.frames-title {
+  height: 90px;
+  font-size: 25px;
+  font-weight: 550;
+  margin-bottom: -60px;
 }
 
-.work > .carttt > .cart > h1 > span {
-  display: block;
+footer {
+  width: 100%;
+  height: 100vh;
+  padding: 120px 24px;
 }
 
-/* about selector */
-.about {
+.icons {
   display: flex;
-  justify-content: center;
   flex-direction: column;
-  padding: 10rem 2rem 6rem 2rem;
-  background: var(--bkcolor);
-  color: var(--txtcolor);
-  background-image: url(Resources/Images/Frame.png);
-  background-repeat: no-repeat;
-  background-position: right 20rem;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  gap: 1.5rem;
```

Comment:
- [ ] Kindly correct the section inside the red lines to be similar to the Figma template. 👇🏽 
- [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.
![image](https://user-images.githubusercontent.com/58642949/187038866-55ca2cb2-e98c-4514-8dd8-d8b8aab2fe80.png)